api/src/main/resources/workbench-api.yaml

```diff
@@ -5707,6 +5799,15 @@ definitions:
     - Deleting
     - Deleted
     - Unknown
+  DiskStatus:
+    type: string
+    enum:
+      - Creating
```

Comment:
we should add an Unknown state on our side. This will be useful if Leo adds new states on us.. Better to return this and log rather than throwing an error when we see an unknown status from leo. You can return this if you see a `null` status from Leo in the mapper (see the runtime status mapper for an example)

Reply:
Maybe I could also add a Conflict state to indicate we've leaked a PD?

Reply:
Probably I'd prefer to just log internally for now. Probably we should only return a different status if it's something the user can act upon or we are unable to continue.